docs/rules/index.md

```diff
@@ -5,42 +5,45 @@ layout: doc
 <!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->
```

Comment:
Sounds like a plan, and I'm happy to try another approach if need be. Just so I'm clear, should I leave the style change in this PR and then open another PR to the ESLint main repository with the markdown changes?